example/30-cloudprofile.yaml

```
@@ -23,6 +23,7 @@ spec:
       expirationDate: 2022-11-30T01:02:03Z # optional
   machineImages:
   - name: suse-chost
+    # updateStrategy: "patch" # optional, either {patch,minor,major}
```

```suggestion
  # updateStrategy: major # optional, either {patch,minor,major}
```

(since this is the default)